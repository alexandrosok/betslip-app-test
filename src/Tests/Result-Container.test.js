import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import {shallowToJson} from 'enzyme-to-json';
import ResultContainer from "../components/Result-Container/result-container";
configure({adapter: new Adapter()})

describe('ResultContainer', () => {
    const component = shallow(<ResultContainer sport="mockSport" eventName="mockEventName"/>);

    it('should render correctly', () => {
        expect(shallowToJson(component)).toMatchSnapshot();
    });
});

describe('ResultContainer containers', () => {
    const component = shallow(<ResultContainer />);

    it('should have a div with class div.result-container', () => {
        expect(component.find('div.result-container').length).toEqual(1)
    })
});

