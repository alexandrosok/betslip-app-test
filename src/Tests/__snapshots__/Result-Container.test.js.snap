// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultContainer should render correctly 1`] = `
<div
  className="result-container"
>
  <div />
  <div>
    <CarouselContainer />
  </div>
</div>
`;
